pipeline/notifications/check_message.py

```diff
@@ -0,0 +1,25 @@
+from typing import List
+
+import prefect
+from prefect import task
+
+from pipeline.types import Notification, User
+
+
+class Notifier:
+    def __init__(self):
+        pass
+
+    def check_msg_notifier(self, notif: Notification) -> List[User]:
+
+        # check if all users in match have a message
+        # if there are users who don't have a message, return a list of them for notification
+        users_with_msg = []
+        users_to_notify = []
+        for msg in notif.chatdata.messages:
+            if msg.from_user in notif.match.users:
```

Comment:
Good thinking. I would have assumed that only users who are in the match could have sent a message to the chat, but this is a good check.

When you refactor to use only `ChatData` instead of both `ChatData` and `Match`, you can check `chatdata.participants` for the user ID.